v8files-extractor.os

```diff
@@ -284,6 +293,7 @@
 
 Процедура ВывестиСправку()
 	Сообщить("Утилита сборки/разборки внешних файлов 1С");
+	Сообщить(ПолучитьВерсию());
```

Comment:
Выводится. Смотри другие изменения 

Отправлено с моего HTC

----- Ответное сообщение -----
От: "Artur Ayukhanov" <notifications@github.com>
Кому: "xDrivenDevelopment/precommit1c" <precommit1c@noreply.github.com>
Копия: "Nikita Gryzlov" <nixel2007@gmail.com>
Тема: [precommit1c] Обновление v8reader (#72)
Дата: вт, марта 22, 2016 21:40

In v8files-extractor.os:
> @@ -284,6 +293,7 @@
>  
>  Процедура ВывестиСправку()
>   Сообщить("Утилита сборки/разборки внешних файлов 1С");
> + Сообщить(ПолучитьВерсию());

@nixel2007 Вывод версии нужен при обычном запуске, а не только в справке/

Добавишь показ версии для обычного запуска?

&mdash;You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub